ui/src/routes/conversations/[id]/+page.svelte

```diff
@@ -190,8 +199,7 @@
 
   function scrollToBottom(delay: number = 0) {
     setTimeout(() => {
-      if (!conversationContainerRef) return;
-      conversationContainerRef.scrollTop = conversationContainerRef.scrollHeight;
+      if (virtualList) virtualList.scrollToBottom();
```

Comment:
```suggestion
      if (!virtualList) return;
      virtualList.scrollToBottom();
```

I prefer this pattern because it makes clear that we *always* expect virtual list to be defined by the time this function is run.